courses/factories.py

```diff
@@ -45,27 +44,38 @@ def _setup_next_sequence(cls):
 
 class CourseRunFactory(DjangoModelFactory):
     """Factory for CourseRuns"""
-    title = fuzzy.FuzzyText(prefix="CourseRun ")
+    title = factory.LazyAttribute(
+        lambda x: "CourseRun " + FAKE.sentence()
+    )
     course = factory.SubFactory(CourseFactory)
     # Try to make sure we escape this correctly
-    edx_course_key = fuzzy.FuzzyText(prefix="course:/()+&/")
-    enrollment_start = fuzzy.FuzzyDateTime(datetime(1980, 1, 1, tzinfo=pytz.utc))
-    start_date = fuzzy.FuzzyDateTime(datetime(1980, 1, 1, tzinfo=pytz.utc))
-    enrollment_end = fuzzy.FuzzyDateTime(datetime(1980, 1, 1, tzinfo=pytz.utc))
-    end_date = fuzzy.FuzzyDateTime(datetime(1980, 1, 1, tzinfo=pytz.utc))
-    fuzzy_start_date = fuzzy.FuzzyText(prefix="Starting ")
-    fuzzy_enrollment_start_date = fuzzy.FuzzyText(prefix="Enrollment starting ")
-    enrollment_url = fuzzy.FuzzyText(prefix="http://")
-    prerequisites = fuzzy.FuzzyText(prefix="CourseRun requires ")
+    edx_course_key = factory.LazyAttribute(
+        lambda x: "course:/()+&/" + FAKE.sentence()
+    )
+    enrollment_start = factory.LazyAttribute(
+        lambda x: FAKE.date_time_this_month(before_now=True, after_now=False, tzinfo=pytz.utc)
```

Comment:
It seems like these changes make sure all course runs are currently running, but I think we want to generate course runs in the past or future too. Though we don't use these values generated by the factory at the moment